src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const paymentPerDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+
+  let totalPrice = paymentPerDay * days;
```

Comment:
You don't really need to change the value of this calculation, add a return statements like `return totalPrice - discount'
```suggestion
  const totalPrice = paymentPerDay * days;
```